sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/ctl/AsyncCtlWorkload.java

```diff
@@ -276,13 +279,23 @@ private void createPrePopulatedDocs(int numberOfPreCreatedDocuments) {
                     PojoizedJson x =
                         resp.getItem();
                     return x;
+                }).onErrorResume(throwable -> {
+                    failureCount.incrementAndGet();
+                    logger.error("Error during pre populating item ", throwable.getMessage());
```

Comment:
logging this as warning instead? Doesn't do any harm, correct?

Reply:
Thanks , that make sense , i will keep it in note and add in it next time when i touch ctl, need this to get merge now